elements/bulbs-video/elements/rail-player/rail-player.js

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes } from 'react';
+import { registerReactElement } from 'bulbs-elements/register';
+import BulbsElement from 'bulbs-elements/bulbs-element';
+
+import VideoField from '../../fields/video';
+import VideoRequest from '../../fields/video-request';
+
+import RailPlayerRoot from './components/root';
+
+import './rail-player.scss';
+
+export default class RailPlayer extends BulbsElement {
+  initialDispatch () {
+    this.store.actions.fetchVideo(this.props.src);
+  }
+
+  componentDidUpdate (prevProps) {
+    if (this.props.src !== prevProps.src) {
+      this.initialDispatch();
+    }
+  }
+
+  render () {
+    return (
+      <RailPlayerRoot
+        {...this.state}
+        muted={typeof this.props.muted === 'string'}
```

Comment:
Re: my other comment, not sure if we even want the muted part to be optional...should always be muted by default